Simplenote/SPShadowView.swift

```diff
@@ -0,0 +1,97 @@
+import UIKit
+
+// MARK: - SPShadowView
+//
+// SPShadowView draws outside shadow, while everyting inside is kept transparent
+//
+final class SPShadowView: UIView {
+    private let cornerRadius: CGFloat
+    private let roundedCorners: UIRectCorner
+    private let maskLayer: CAShapeLayer = CAShapeLayer()
+
+    override var bounds: CGRect {
+        didSet {
+            updatePath()
+        }
+    }
+
+    /// Designated Initializer
+    ///
+    ///  - cornerRadius: The radius of each corner oval
+    ///  - roundedCorners: A bitmask value that identifies the corners that you want rounded
+    ///
+    init(cornerRadius: CGFloat, roundedCorners: UIRectCorner) {
+        self.cornerRadius = cornerRadius
+        self.roundedCorners = roundedCorners
+
+        super.init(frame: .zero)
+        configure()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - Private Methods
+//
+private extension SPShadowView {
+
+    /// Initial configuration of the view
+    ///
+    func configure() {
+        backgroundColor = .clear
+
+        configureShadow()
+        configureMask()
+
+        updatePath()
+    }
+
+    /// Configuration of the shadow
+    ///
+    func configureShadow() {
+        layer.shadowColor = UIColor.black.cgColor
```

Comment:
Maybe it'd be useful to parametrize this one? 🤔 

(In the lines of `var shadowColor = UIColor.black`?)

Reply:
Good point